src/ajax/ajax.js

```diff
@@ -43,10 +44,33 @@ define([ "shoestring" ], function(){
 			return;
 		}
 
+		// create parameter string from data object
+		if( settings.data !== null ){
+			var params = "";
+			for( var key in settings.data ){
+				if( settings.data.hasOwnProperty( key ) ){
+					if( params !== "" ){
+						params += "&";
+					}
+					params += encodeURIComponent( key ) + "=" + 
+						encodeURIComponent( settings.data[key] );
+				}
+			}
+		}
+
+		// append params to url for GET requests
+		if( settings.method === "GET" && params ){
+			url += "?" + params;
```

Comment:
his seems to assume that there are no params attached to the url but I don't think we strip them or otherwise enforce this restriction on the url elsewhere.